code/game/world.dm

```diff
@@ -299,4 +299,7 @@ GLOBAL_LIST_EMPTY(world_topic_handlers)
 	rustg_close_async_http_client() // Close the HTTP client. If you dont do this, youll get phantom threads which can crash DD from memory access violations
 	disable_auxtools_debugger() // Disables the debugger if running. See above comment
 	rustg_redis_disconnect() // Disconnects the redis connection. See above.
+	#ifdef ENABLE_BYOND_TRACY
+	CALL_EXT("prof.dll", "destroy")() // Setup Tracy integration
```

Comment:
And yes I know these should probably be cross platform.

Guess what. We dont run on linux. We need basic linux support for CI stuff, not for experimental performance memes.